packages/next/lib/is-error.ts

```diff
@@ -11,3 +13,22 @@ export default function isError(err: unknown): err is NextError {
     typeof err === 'object' && err !== null && 'name' in err && 'message' in err
   )
 }
+
+export function getProperError(err: unknown): Error {
+  if (isError(err)) {
+    return err
+  }
+
+  if (process.env.NODE_ENV === 'development') {
+    // provide better error for case where `throw undefined`
+    // is called in development
+    if (typeof err === 'undefined') {
```

Comment:
What about null?

Reply:
Was wondering if we should add a case for null here as well, seems to good to have. Added in https://github.com/vercel/next.js/pull/33159/commits/c0dae27db93b330c33e4d47076cd3d6cbf07d1fc